upstream-community-operators/seldon-operator/seldonoperator.crd.yaml

```diff
@@ -12,5 +12,5 @@ spec:
     - sdep
   scope: Namespaced
   version: 0.1.2
-  group: machinelearning.seldon.io
+  group: seldon
```

Comment:
spec:group: The API group should reflect the collection of logically related objects. For example, all batch objects, such as Job or ScheduledJob, can belong to the batch API Group, such as batch.kyma-project.io. As best practice, use the fully-qualified domain name of the organization (kyma-project.io) preceded by a subgroup if necessary, for example crontab.kyma-project.io. The group name should reflect a capability-related and not an implementation-related name. For example, for eventing use eventing.kyma-project.io and not nats.kyma-project.io. Avoid prefixing the name with more than a subgroup, like in this example: user.crontab.kyma-project.io. If the subgroup consists of multiple words, do not use spaces, hyphens, or CamelCase.